nixos/modules/services/networking/lxd-image-server.nix

```
@@ -0,0 +1,144 @@
+{ config, pkgs, lib, ... }:
+
+with lib;
+
+let
+  cfg = config.services.lxd-image-server;
+  format = pkgs.formats.toml {};
+
+  location = "/var/www/simplestreams";
+in
+{
+  options = {
+    services.lxd-image-server = {
+      enable = mkEnableOption "lxd-image-server";
+
+      group = mkOption {
+        type = types.str;
+        description = "Group assigned to the user and the webroot directory.";
+        default = "nginx";
+        example = "www-data";
+      };
+
+      extraConfig = mkOption {
```

```suggestion
      settings = mkOption {
```

[RFC42](https://github.com/NixOS/rfcs/blob/master/rfcs/0042-config-option.md) suggests calling this `settings` instead.